### ISSUE TITLE
Oops! The 'bcftools' MultiQC module broke

### ISSUE THREAD

**Issue by jamigo** [ewels/MultiQC#1400]
**Description of bug:**
[ERROR ] Oops! The 'bcftools' MultiQC module broke...

**MultiQC Error log:**

```
============================================================
Module bcftools raised an exception: Traceback (most recent call last):
  File "/home/user/.local/lib/python3.6/site-packages/multiqc/multiqc.py", line 579, in run
    output = mod()
  File "/home/user/.local/lib/python3.6/site-packages/multiqc/modules/bcftools/bcftools.py", line 39, in __init__
    n["stats"] = self.parse_bcftools_stats()
  File "/home/user/.local/lib/python3.6/site-packages/multiqc/modules/bcftools/stats.py", line 114, in parse_bcftools_stats
    quality = float(s[2].strip())
ValueError: could not convert string to float: '.'
============================================================
```

**File that triggers the error:**

content of the file example.vcf.stats.txt

```
# QUAL	[2]id	[3]Quality	[4]number of SNPs	[5]number of transitions (1st ALT)	[6]number of transversions (1st ALT)	[7]number of indels
QUAL	0	.	76	38	38	19
```

**MultiQC run details (please complete the following):**

* Command used to run MultiQC: `/home/user/.local/bin/multiqc -f --no-data-dir -c file.yaml`
* MultiQC Version: MultiQC v1.10
* Operating System:  RHELS 7.5
* Python Version: Python 3.6
* Method of MultiQC installation: pip

**Additional context**

Using bcftools version 1.12 there seems to appear "." values in the [3]Quality column, which are not properly parsed by the bcftools module. I've worked around this issue by changing those "." values to "0.0", but a proper solution should be provided.

**Comment by ewels** [comment]
Should be a fairly easy PR to catch this exception type and assign the value to 0, if you fancy having a go at it...

**Comment by BiscuitCandy** [comment]
i have done it
mind checking, someone
PR : solved breakdown bcftools #1402
